Automattic/jetpack
# Akismet/Anti-spam: always use Akismet logo in plugin header

## Proposed changes:
* The standard Akismet logo will be shown for everyone using the Akismet plugin. Currently, we show the Jetpack Anti-spam logo for Jetpack Anti-spam subscribers (since https://github.com/Automattic/jetpack/pull/31464), which has caused some confusion.

The following plugin walkthrough posts identified this as an issue: pfln9p-Ei-p2#comment-766 and pekYwv-4oE-p2#comment-3814.

### Other information:


## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
On a Jetpack development site, try signing up for a Jetpack Anti-Spam subscription and then visit /wp-admin/admin.php?page=akismet-key-config. You should see the normal Akismet logo.

<img width="772" alt="Screenshot 2024-08-23 at 15 46 26" src="https://github.com/user-attachments/assets/d3be5c64-a6d8-407e-84aa-98ae2ddbc8e7">
